Sources/JavaScriptKit/Support.swift

```diff
@@ -0,0 +1,57 @@
+protocol _AnyJSValueCodable: JSValueConvertible {}
+
+public struct AnyJSValueCodable: JSValueCodable, ExpressibleByNilLiteral {
```

Comment:
This is necessary for things like [`AddEventListenerOptions`](https://github.com/j-f1/DOMKit/blob/main/Sources/DOMKit/WebIDL/AddEventListenerOptions.swift).